cylc/flow/etc/tutorial/runtime-tutorial/flow.cylc

```diff
@@ -6,9 +6,9 @@
     # Start the workflow 7 hours before now ignoring minutes and seconds
     # * previous(T-00) takes the current time ignoring minutes and seconds.
     # * - PT7H subtracts 7 hours from the time.
-    initial cycle point = previous(T-00) - PT7H
+    initial cycle point = # Change me
     # Stop the workflow 6 hours after the initial cycle point.
-    final cycle point = +PT6H
+    final cycle point = # Change me
```

Comment:
Looking back at the [Cylc 7 tutorial(https://github.com/metomi/rose/blob/caafa20cb659d3c90000215ddcbc08903a20e211/etc/tutorial/runtime-tutorial/suite.rc#L5-L6) this was supposed to be a couple of TODO comments.

```suggestion
    # TODO - set the initial cycle point.
    # TODO - set the final cycle point.
```